src/main/java/org/killbill/billing/payment/api/PaymentApi.java

```diff
@@ -646,4 +646,13 @@ public void setDefaultPaymentMethod(Account account, UUID paymentMethodId, Itera
     @RequiresPermissions(PAYMENT_METHOD_CAN_UPDATE)
     public List<PaymentMethod> refreshPaymentMethods(Account account, Iterable<PluginProperty> properties, CallContext context)
             throws PaymentApiException;
+
+    /**
+     * @param paymentTransactionId the payment transaction id
+     * @param context         the call context
+     * @return the payment transaction
+     * @throws PaymentApiException
+     */
+    public PaymentTransaction getPaymentTransactionById(final UUID paymentTransactionId, final TenantContext context)
+            throws PaymentApiException;
 }
```

Comment:
Yes you are right.
I thought we needed the PaymentTransaction instead, but I can change that to get the Payment.
Working on that! 📝
Thanks!